src/configManager.ts

```diff
@@ -25,12 +25,10 @@ export const storeConfig = (options: SdkOptions, productDetails: ProductDetails)
   }
 };
 
-export const loadConfig = (
-  options: SdkOptions,
-  productDetails: ProductDetails
-): storedConfig | null => {
-  if (options.useCookie) {
-    return loadConfigFromCookie(productDetails);
```

Comment:
removed options arg so we can use this in bootstrapping when we don't have options available